Assets/Infiltration/Scripts/UIManager.cs

```diff
@@ -0,0 +1,51 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Infiltration
+{
+    public class UIManager : MonoBehaviour
+    {
+        private bool _stateGame;
+        public static bool StateSet;
```

Comment:
Consider using a ```GameManager``` to centralize game running state, scene loading and event dispatching if needed.